content/sections/pros.en.md

```diff
@@ -6,29 +6,33 @@ weight = 1
 
 type = "pros"
 
-using_another_system = "Using another system"
+using_another_system = "Using another system?"
 contact_us = "contact us"
 
 
 [[paragraphs]]
-title = "A ready-to-use platform"
+title = "Online Catalogue with checkout"
 content = """
-    We are offering you more than a simple online showcase. Choose between our attractive canvases specifically \
-    designed for optometrists, and we will help you integrate your branding as well as your products and services.
+The Internet has revolutionized the way people shop. Give customers the experience they are looking for and attract new customers along the way with your online catalogue. One they can see when it is convenient for them and from where it is convenient for them.
```

Comment:
Sentences added by Doc after my review! I will remove it. Why use line breaks? @osylvain